modules/check_in/app/docs/check_in/v2/check_in.yaml

```diff
@@ -284,6 +284,56 @@ paths:
                 data:
                   error: true
                   message: 'Internal service error message'
+  /pre_check_ins:
+    post:
+      tags:
+        - Create PreCheck-in
+      summary: Confirm Pre Check-in data
+      operationId: create_pre_check_in
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreatePreCheckInBody'
+      responses:
+        '200':
+          description: Pre Check-in data was confirmed successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SuccessfulPreCheckIn'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Unauthorized'
+              example:
+                status: 401
+                data:
+                  error: true
+                  message: 'Token verification failed: error message'
```

Comment:
nit: `unauthorized_message` from `Session` object seem to be returning `{ permissions: 'read.none', status: 'success', uuid: uuid }`

Reply:
updated in 2f84e5f50a810f003c289662a79e79a1d922b0ed